doc/source/tutorials/bipartite_matching_maxflow/bipartite_matching_maxflow.rst

```diff
@@ -0,0 +1,89 @@
+.. include:: include/global.rst
+
+.. tutorials-bipartite-matching-maxflow
+
+==========================================
+Maximum Bipartite Matching by Maximum Flow
+==========================================
+
+This example presents how to visualise bipartite matching using maximum flow. Please note that the *igraph* already has :meth:`maximum_bipartite_matching` which is better suited for finding the maximum bipartite matching. For a demonstration on how to use that method instead, check out `Maximum Bipartite Matching<>`_. This particular example is purely for demonstrative purposes.
+
+.. TODO: add link to Maximum Bipartite Matching
+
+.. code-block:: python
+
+    import igraph as ig
+    import matplotlib.pyplot as plt
+
+    # Generate the graph
+    g = ig.Graph(
+        9,
+        [(0, 4), (0, 5), (1, 4), (1, 6), (1, 7), (2, 5), (2, 7), (2, 8), (3, 6), (3, 7)],
+        directed=True
+    )
+
+    # Assign nodes 0-3 to one side, and the nodes 4-8 to the other side
+    for i in range(4):
```

Comment:
Another way of saying this: `g.vs["type"] = [vertex.index < 4 for vertex in g.vs]`; again, I'm okay with an explicit for loop but just wanted to make sure that you are aware of this.